popup.js

```diff
@@ -82,16 +82,19 @@ document.getElementById('convertto').onchange=function (e) {
 	});
 };
 
-document.getElementById('sendto').onchange=function (e) {
-	var cburl = e.target.options[e.target.selectedIndex].text;
+
```

Comment:
the same with these changes: not necessary, undo them in the new commit.

Please follow our formatting guidelines: For javascript, all commands end with a semi-colon, even if it is acceptable to not be there by the language, we do not accept it.